.env.development

```diff
@@ -5,7 +5,10 @@ REACT_APP_MESSAGING_ROOT=https://dev-k8s.treetracker.org/messaging
 REACT_APP_QUERY_API_ROOT=https://dev-k8s.treetracker.org/query
 REACT_APP_REPORTING_API_ROOT=https://dev-k8s.treetracker.org/reporting
 REACT_APP_TREETRACKER_API_ROOT=https://dev-k8s.treetracker.org/treetracker
+REACT_APP_API_URL=https://dev-k8s.treetracker.org
+REACT_APP_EARNINGS_API_MAPPING=/earnings
 REACT_APP_ENABLE_CAPTURE_MATCHING=true
 REACT_APP_ENABLE_EARNINGS=true
 REACT_APP_ENABLE_PAYMENTS=true
 REACT_APP_ENABLE_MESSAGING=true
+REACT_APP_ENABLE_STAKEHOLDERS=true
```

Comment:
I would suggest adding `REACT_APP_STAKEHOLDER_API_ROOT=https://dev-k8s.treetracker.org/stakeholder/stakeholders` to `.env.development`. It might save time for people who are looking to build on top of your changes locally (it took me a while to realize the microservice lived at `/stakeholder/stakeholders` and not `/stakeholders` or `/stakeholder`. Looks like Nick mentioned this in another [PR comment](https://github.com/Greenstand/treetracker-admin-client/pull/262#discussion_r785471670)

Reply:
Done!

Reply:
Done!